spec/models/student_spec.rb

```diff
@@ -4,4 +4,18 @@
   describe 'relationships' do
     it { should belong_to :enrollment }
   end
+
```

Comment:
These tests would probably be better suited utilizing FactoryBot

Reply:
Technically, no. Tests that are best suited for FactoryBot would be ones involving having to create A LOT of instances of a class/model.

Since this test only needs two students then it's not "better suited for Factory Bot" but do the switch for consistency's sake if you'd like to.